integration/onewatch/start.sh

```diff
@@ -29,8 +29,7 @@ while true; do
     "$@" &
     process_id=$!
     echo "$process_id" > process.txt
-    wait $process_id
-    EXIT_CODE=$?
+    wait $process_id && EXIT_CODE=$? || EXIT_CODE=$?
```

Comment:
when we killed the process to restart, `wait` returned a non-0 exit, so the script exited because of `set -e`

Reply:
would it make sense to just `set +e` before the `wait`?